### ISSUE TITLE
One liner to validate ISBN 13

### ISSUE THREAD

**Issue by gotson** [SaekiTominaga/isbn-verify#1]
Hi,

thanks for the nice library, however whenever i want to check if an ISBN is valid and ISBN 13 i have to do:

```ts
const isbn = new ISBN(value)
return isbn.isIsbn13() && isbn.isValid())
```

This is quite verbose, and since I have to instantiate a variable, I can't write a one-liner for validation functions.

For example in Vuelidate, when checking if a field is either empty or a valid ISBN 13:

```ts
const validIsbn = (value: any) => {
  const isbn = new ISBN(value)
  return (!helpers.req(value) || (isbn.isIsbn13() && isbn.isValid()))
}
```

It would be nice to have helper functions that would check validity and ISBN 13 at the same time. Something like: `isValidIsbn13(string)`

That way I could write:

```ts
const validIsbn = (value: any) => !helpers.req(value) || isValidIsbn13(value)
```

Which would be much more concise!

**Comment by SaekiTominaga** [comment]
Hi, @gotson.
Would adding arguments to the current method, rather than adding a new method, accomplish what you want?

- `isIsbn13()` → Only check if the formats match.
- `isIsbn13(true)` or `isIsbn13( { check_digit: true } )` → Checks if the format matches and the check digit is correct.

 For example, it could look like this.
```ts
import IsbnVerify from '@saekitominaga/isbn-verify';
const validIsbn = (value: any) => !helpers.req(value) || new IsbnVerify(value).isIsbn13( { check_digit: true } )
```

**Comment by gotson** [comment]
Or simpler, add a function `isIsbn13Valid()` ? Could also add `isIsbn10Valid()` but i don't use isbn10 :)

**Comment by gotson** [comment]
Yes it would, but i think it would make the prototypes more complex. Up to you, it's your code and library, i would be happy both ways :)

**Comment by SaekiTominaga** [comment]
Released as [v1.4.0](https://www.npmjs.com/package/@saekitominaga/isbn-verify/v/1.4.0).

As mentioned in the previous comment, it was supported by adding an argument to the existing method.
This is a judgment that takes into consideration further expandability in the future and the evolution of ISBN itself (the possibility of increasing the number of digits further).
Thank you.
